build.gradle

```diff
@@ -13,7 +13,7 @@ tasks.withType(JavaCompile) {
 
 wrapper{
     distributionType = Wrapper.DistributionType.ALL
-    gradleVersion = '8.6'
+    gradleVersion = '8.12'
```

Comment:
Can we bump to 8.13?

Reply:
This breaks the build and I did not want to spend time on figuring out why. I suggest doing it in a separate PR (i.e. "next time something breaks").